tests/operator_test.go

```diff
@@ -395,6 +396,12 @@ var _ = Describe("Operator", func() {
 	}
 
 	ensureShasums := func() {
+		if tests.SkipShasumCheck {
+			// Currently shasums are not supported on crio.
+			log.Log.Warning("Cannot use shasums when using CRI-o")
```

Comment:
this still mentions cri-o. We just want to skip shasums test, and don't care why imho